internal/lookout/ui/src/containers/JobSetsContainer.tsx

```diff
@@ -266,8 +245,44 @@ class JobSetsContainer extends React.Component<JobSetsContainerProps, JobSetsCon
     })
   }
 
-  private fetchJobSets(getJobSetsRequest: GetJobSetsRequest): Promise<JobSet[]> {
-    return this.props.jobService.getJobSets(getJobSetsRequest)
+  private async fetchJobSets(getJobSetsRequest: GetJobSetsRequest): Promise<JobSet[]> {
+    const response = await this.props.v2GroupJobsService.groupJobs(
+      [
+        {
+          isAnnotation: false,
+          field: "queue",
+          value: getJobSetsRequest.queue,
+          match: Match.Exact,
+        },
+      ],
+      getJobSetsRequest.activeOnly,
+      {
+        field: "submitted",
+        direction: getJobSetsRequest.newestFirst ? "DESC" : "ASC",
+      },
+      {
+        field: "jobSet",
+        isAnnotation: false,
+      },
+      ["state", "submitted"],
+      0,
+      0,
+    )
+
+    return response.groups.map((group) => {
+      const state = group.aggregates.state as Record<string, number>
+      return {
+        jobSetId: group.name,
+        queue: getJobSetsRequest.queue,
+        jobsQueued: state[JobState.Queued] || 0,
+        jobsPending: state[JobState.Pending] || 0,
+        jobsRunning: state[JobState.Running] || 0,
+        jobsSucceeded: state[JobState.Succeeded] || 0,
+        jobsFailed: state[JobState.Failed] || 0,
+        jobsCancelled: state[JobState.Cancelled] || 0,
+        latestSubmissionTime: group.aggregates.submitted as string,
+      }
+    })
```

Comment:
This is the real change: we now retrieve the status counts from version 2 of the Lookout API.